iOS/ViewControllers/Courses/CourseItemViewController.swift

```diff
@@ -29,8 +29,13 @@ class CourseItemViewController: UIPageViewController {
             self.nextItem = self.currentItem?.nextItem
 
             if let item = self.currentItem, let section = item.section {
-                self.progressLabel.text = "\(item.position) / \(section.items.count)"
+                let sortedCourseItems = section.items.sorted(by: \.position)
+                if let index = sortedCourseItems.firstIndex(of: item) {
+                let position = index + 1
+
+                self.progressLabel.text = "\(position) / \(section.items.count)"
                 self.progressLabel.sizeToFit()
+                }
```

Comment:
We should also handle the case if the item can't be found in the `sortedCourseItems`. I would prefer something like this: `- / 14`.